test/common/typeconfig.go

```diff
@@ -21,13 +21,12 @@ import (
 
 	"github.com/kubernetes-sigs/federation-v2/pkg/apis/core/typeconfig"
 	fedv1a1 "github.com/kubernetes-sigs/federation-v2/pkg/apis/core/v1alpha1"
-	"sigs.k8s.io/controller-runtime/pkg/client"
+	client "github.com/kubernetes-sigs/federation-v2/pkg/client/generic"
```

Comment:
(No action required) Was this change required for the transition to kubebuilder?  It's nice to have, don't get me wrong, but I'd prefer to review PRs with a cohesive purpose.

Reply:
After transitioning, i started getting errors like the API type not found. The generic client uses a scheme which includes all the types from federation, k8s and cluster-registry. So replacing the client solved the issue.